test/.package.json

```diff
@@ -11,9 +11,9 @@
   "license": "Apache-2.0",
   "scripts": {
     "test:pass": "echo 'Expecting pass' && eslint pass.jsx",
-    "test:fail-jsdoc": "echo 'Expecting failure' && eslint jsdoc-fail.js; if [ $? -eq 1 ]; then exit 0; fi",
```

Comment:
If `eslint jsdoc-fail.js` fails (as expected) then we will exit 0, which is the intended behaviour.
But suppose the exit code from `eslint jsdoc-fail.js` is 0. Surely then we need to fail, as eslint didn't detect the error we expected it to?